eng/Versions.props

```diff
@@ -4,7 +4,7 @@
     <MicrosoftWindowsCsWinRTPackageVersion>1.6.1</MicrosoftWindowsCsWinRTPackageVersion>
     <CsWinRTDependencyDotNetCoreSdkPackageVersion>5.0.404</CsWinRTDependencyDotNetCoreSdkPackageVersion>
     <CsWinRTDependencyDotNetCoreRuntimePackageVersion>5.0.13</CsWinRTDependencyDotNetCoreRuntimePackageVersion>
-    <CsWinRTDependencyWindowsSdkPackagePackageVersion>24</CsWinRTDependencyWindowsSdkPackagePackageVersion>
+    <CsWinRTDependencyWindowsSdkVersionSuffixPackageVersion>24</CsWinRTDependencyWindowsSdkVersionSuffixPackageVersion>
```

Comment:
If you are changing the name here, I think you'd also need to change it in version.details.xml

Reply:
Thanks, addressed.